components/tag-list/tag-list.js

```diff
@@ -297,7 +298,7 @@ class TagList extends LocalizeCoreElement(InteractiveMixin(ArrowKeysMixin(LitEle
 
 			return {
 				isHidden: computedStyles.display === 'none',
-				width: Math.ceil(parseFloat(computedStyles.width) || 0)
```

Comment:
The calculations become more exact so removing the `Math.ceil` in a couple places